Source/CarthageKit/Dependency.swift

```diff
@@ -151,6 +158,26 @@ extension Dependency: Scannable {
 			return .failure(ScannableError(message: "empty string after dependency type", currentLine: scanner.currentLine))
 		}
 	}
+
+	// MacOS 10.10 Compatibility Helper
```

Comment:
Maybe we should just drop 10.10 at this point? 🤔

Reply:
Yeah :D I think 10.11 is acceptable

Reply:
Woohoo! That makes this way easier. I played around this for way to long learning about the intricacies of URL comparisons.